rootValue/rootValue.js

```diff
@@ -34,8 +34,8 @@ module.exports = RootValue = {
   sendEmail: async (args, req) => {
     try {
       const isAuth = await req.isAuth;
-      if (!isAuth) throw new Error('Please autheticate');
-      console.log('Yaha aako xa ra??', req.userId);
+      if (!isAuth) throw new Error("Please autheticate");
+      console.log("Yaha aako xa ra??", req.userId);
```

Comment:
Is the user/developer always going to be Nepali?

Reply:
Comments has been deleted.